stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1103,7 +1103,7 @@ static void traceJobQueue(DefaultActorImpl *actor, Job *first) {
 static SWIFT_ATTRIBUTE_ALWAYS_INLINE void traceActorStateTransition(DefaultActorImpl *actor,
     ActiveActorStatus oldState, ActiveActorStatus newState) {
 
-  SWIFT_TASK_DEBUG_LOG("Actor %p transitioned from %#x to %#x (%s)\n", actor,
+  SWIFT_TASK_DEBUG_LOG("Actor %p transitioned from %#x to %#x (%s)", actor,
```

Comment:
Spurious change?

Reply:
Flyby cleanup; these messages should not have \n since the function adds it by itself so this gave a weird empty line between logs -- happy to do in separate Pr tho if you prefer